scripts/ci-frontend-metrics.sh

```diff
@@ -11,6 +11,7 @@ ACCESSIBILITY_ERRORS="$(grep -oP '\"errors\":(\d+),' pa11y-ci-results.json | gre
 DIRECTIVES="$(grep -r -o  directive public/app/ | wc -l)"
 CONTROLLERS="$(grep -r -oP 'class .*Ctrl' public/app/ | wc -l)"
 LEGACY_FORMS="$(grep -r -oP 'LegacyForms;' public/app | wc -l)"
+BARREL_IMPORTS="$(grep -r -oP '@todo: replace barrel import path' public/app | wc -l)"
```

Comment:
not sure if it helps, but any rule you add to betterer will automatically output metrics anyway (see lines 44-49). so that might make this unnecessary?

Reply:
I think the betterer output will relate to the creation of barrel files. This instead should help us understand where those barrel files are consumed.